kotlintest-assertions/src/commonMain/kotlin/io/kotlintest/matchers/string/matchers.kt

```diff
@@ -281,3 +281,22 @@ fun beEqualIgnoringCase(other: String) = neverNullMatcher<String> { value ->
     "${value.show()} should not be equal ignoring case ${other.show()}"
   )
 }
+
+fun String?.shouldBeTruthy() = this should beTruthy()
```

Comment:
We should probably add some kdocs to this.
If you want, I can do it for you :D

Reply:
No problem, I will do it